README.md

```diff
@@ -113,6 +113,9 @@ To fetch a specific service, for example CampaignService:
 See the provided [examples][] for more detailed demonstrations of how to use the
 library.
 
+**Note:** From the source code, you can ignore comments with `[START...]` and
```

Comment:
Can we reword as "You can ignore comments with .... in examples—they are specifically used by Google." I think that flows better.

Also, can we be any more specific about what we're using it for? It's not malicious, so being more open is probably better.

Reply:
I reworded as suggested, thanks!

When it comes to adding extra details about what these are used for though, we discussed that with Q & Nick and we decided to not do it. Let's discuss this offline if you strongly believe that it is required.